crates/runtime/src/accelerated_table.rs

```diff
@@ -372,16 +371,44 @@ impl AcceleratedTable {
                     }
                 }
                 AccelerationRefreshMode::Full(receiver) => {
+                    let filter_converter = match (refresh.period, refresh.time_column) {
+                        (None, Some(_) | None) => {
+                            None
+                        },
+                        (Some(_), None) => {
+                            tracing::warn!("[refresh] No time_column is provided, refresh_period will be ignored");
+                            None
+                        }
+                        (Some(_), Some(column)) => {
+                            let schema = federated.schema();
+                            let field = schema.column_with_name(column.as_str());
+                            TimestampFilterConvert::create(field, Some(column), refresh.time_format)
+                        }
+                    };
```

Comment:
This feels unnecessarily complex. Is there a simpler way to achieve what we want?

Reply:
it might be. The main complication comes from informative information to the user if they have set `refresh_period` and expect it is working.

Reply:
That type of validation/warning should happen when we are loading the spicepod imo